src/Resources/config/services.yml

```diff
@@ -36,6 +36,15 @@ services:
             - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest, priority: 30 }
             - { name: kernel.event_listener, event: kernel.finish_request, method: onKernelFinishRequest, priority: -254 }
 
+    contao.listener.exception:
+        class: Contao\CoreBundle\EventListener\ExceptionListener
+        tags:
+            - { name: kernel.event_listener, event: kernel.exception, method: onKernelException }
+        arguments:
+            - "%contao.pretty_error_screens%"
+            - "@templating"
```

Comment:
As both https://github.com/contao/standard-edition/blob/b0f82c6fb3210c40e47135b4d83d01ffad777dcb/composer.json#L19
and https://github.com/contao/core-bundle/blob/f7061486148e0c9b1ba9db72988912986133ee1e/composer.json#L21
are still depending on `~2.6` I'd keep it like it is.
Can be evaluated and changed later on - I simply currently lack the time to read this up.